src/main/java/org/example/cucumber/definitionSteps/MyStepdefs.java

```diff
@@ -0,0 +1,26 @@
+package org.example.cucumber.definitionSteps;
+
+import io.cucumber.java.en.Given;
+import org.example.setUp.testng.BaseTest;
+import org.openqa.selenium.JavascriptExecutor;
+
+public class MyStepdefs extends BaseTest {
+    @Given("something")
+    public void something() {
+
+
+        // Navigate to the desired web page
+        driver.get("https://reportportal.epam.com/");
+
+        // Execute JavaScript code to set Authorization header
+        String token = "Zg3XqEDeYWM9pVGiNnThawaBTBbv5bpY_SS0aSFIaR126ykBZXKxc86JfAw-DA9sThbHMUpcMojSoiA0cnXY4iHhz8eMefX1G";
```

Comment:
hardcode token in the code bad practice . Please fine another solution

Reply:
I deleted this logic because now I don't use it